src/components/PopupWithForm.js

```diff
@@ -23,4 +23,13 @@ export default class PopupWithForm extends Popup {
       this.close();
     });
   }
```

Comment:
**COULD BE IMPROVED**

You can make a special method for getting a form element from `PopupWithForm` not to **search** it in `index.js`



```js

  getForm() {
    return this._popupForm
  }


```